src/main/java/org/folio/entlinks/service/InstanceLinkService.java

```diff
@@ -41,6 +45,29 @@ public void updateInstanceLinks(UUID instanceId, @NotNull InstanceLinkDtoCollect
     repository.saveAll(linksToCreate);
   }
 
+  @Transactional
```

Comment:
I don't think we need transaction on the method

Reply:
right, it is only read. removed